cache/clock_cache.cc

```diff
@@ -714,13 +739,14 @@ class ClockCache final : public ShardedCache {
 
 }  // end anonymous namespace
 
-std::shared_ptr<Cache> NewClockCache(size_t capacity, int num_shard_bits,
-                                     bool strict_capacity_limit) {
+std::shared_ptr<Cache> NewClockCache(
+    size_t capacity, int num_shard_bits, bool strict_capacity_limit,
+    CacheMetadataCharge metadata_charge_policy) {
```

Comment:
A "policy" is typed "charge", which is confusing to me. Is there a way to improve the naming. Say, renaming CacheMetadataCharge to ChargePolicy, ir something like that/

Reply:
I am changing it to CacheMetadataChargePolicy to keep CacheMetadata prefix.